projects/1.16/assets/materialis/materialis/lang/zh_cn.json

```
@@ -0,0 +1,259 @@
+  "item.materialis.molten_quicksilver_bucket": "熔融水银桶",
+  "item.materialis.molten_refined_glowstone_bucket": "熔融强化荧石桶",
+  "item.materialis.molten_refined_obsidian_bucket": "熔融强化黑曜石桶",
+  "item.materialis.molten_refined_radiance_bucket": "液化光辉桶",
+  "item.materialis.molten_regalium_bucket": "熔融贵豪桶",
+  "item.materialis.molten_shadow_steel_bucket": "熔融暗影钢桶",
+  "item.materialis.molten_starmetal_bucket": "熔融星辉桶",
+  "item.materialis.molten_utherium_bucket": "熔融御腐桶",
+  "item.materialis.pink_slime_crystal": "粉色黏液晶体",
+  "item.materialis.wrench": "扳手",
+  "item.materialis.wrench.description": "扳手是工程师常用的工具。它可以有效地拆除机器并且拥有更多的能力槽。\n右键单击: 旋转方块",
+  "item.materialis.wrench_head": "扳手头",
+  "item.materialis.wrench_head_cast": "扳手头黄金铸模",
+  "item.materialis.wrench_head_red_sand_cast": "扳手头红沙铸模",
+  "item.materialis.wrench_head_sand_cast": "扳手头沙子铸模",
+  "material.materialis.aluminum": "铝",
+  "material.materialis.aluminum.encyclopedia": "每级-10%攻击伤害，+20%攻击速度",
+  "material.materialis.arcane_gold": "秘金",
+  "material.materialis.arcane_gold.encyclopedia": "每级+1魔法伤害",
+  "material.materialis.brass": "黄铜",
+  "material.materialis.brass.encyclopedia": "每级+1攻击力，随着耐久度的降低而降低",
+  "material.materialis.brass.flavor": "你喜欢黄铜吗？",
+  "material.materialis.cloggrum": "扼塞",
+  "material.materialis.cloggrum.encyclopedia": "工具的耐久度越高，维修所花费的越少",
+  "material.materialis.ebony_psimetal": "乌金色Psi金属",
+  "material.materialis.ebony_psimetal.encyclopedia": "最多可增加1点额外伤害，随着每级Psi的数量而变化",
+  "material.materialis.fairy": "仙子",
+  "material.materialis.fairy.encyclopedia": "损失生命值的百分比乘以属性等级会加成到采掘速度中",
+  "material.materialis.fairy.flavor": "自地狱深处的仙境而来。",
+  "material.materialis.forgotten_metal": "遗忆",
+  "material.materialis.forgotten_metal.encyclopedia": "每级对深暗之园的生物造成50%额外伤害，同时每级对深暗之园的方块提高25%采掘速度",
+  "material.materialis.forgotten_metal.flavor": "这啊，这是。。。这是那啥，那啥来着。",
+  "material.materialis.forgotten_metal.format": "遗忆 %s",
+  "material.materialis.froststeel": "霜钢",
+  "material.materialis.froststeel.encyclopedia": "每级对目标造成7.5秒的减速效果",
+  "material.materialis.iesnium": "艾瑟",
+  "material.materialis.iesnium.encyclopedia": "允许该工具采掘彼岸矿物",
+  "material.materialis.invar": "殷钢",
+  "material.materialis.invar.encyclopedia": "扑灭目标身上的火焰，并以每1.5秒的燃烧时间乘以属性等级造成1额外伤害",
+  "material.materialis.ivory_psimetal": "象白牙Psi金属",
+  "material.materialis.ivory_psimetal.encyclopedia": "最多可增加1个额外的采掘速度，可根据每级的Psi数量进行调整",
+  "material.materialis.neptunium": "海王",
+  "material.materialis.neptunium.encyclopedia": "对水栖生物造成1.5倍伤害，在水下时采掘速度提高5倍",
+  "material.materialis.nickel": "镍",
+  "material.materialis.nickel.encyclopedia": "每级对正在燃烧的目标增加0.5秒的燃烧时间",
+  "material.materialis.osmium": "锇",
+  "material.materialis.osmium.encyclopedia": "每级增加0.5攻击伤害乘以已损失的生命值百分比的额外伤害",
+  "material.materialis.pewter": "白镴",
+  "material.materialis.pewter.encyclopedia": "每级-10%攻击速度，+20%攻击伤害",
+  "material.materialis.pink_slime": "粉色黏液",
+  "material.materialis.pink_slime.encyclopedia": "每级降低10%的采掘速度和攻击速度，并获得300乘以消耗的数据作为黏液覆层的额外容量",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "material.materialis.pink_slime.encyclopedia": "每级降低10%的采掘速度和攻击速度，并获得300乘以降低的数值的黏液覆层额外容量",
```
这个黏液覆层额外容量是什么东西？还有你确定是“降低”吗？最好进游戏测试一下。